build.py

```diff
@@ -742,9 +766,7 @@ def create_dockerfile_linux(ddir, dockerfile_name, argmap, backends, repoagents,
 COPY --chown=1000:1000 --from=tritonserver_build /workspace/build/sagemaker/serve /usr/bin/.
```

Comment:
There is no "tritonserver_build" when running from compose.py. So you likely need to move the sagemaker out of the common function and modify it as needed in compose.py.